test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -451,5 +477,92 @@ private static void DeleteDirectory(string path)
 
             }
         }
+
+        //temporary added methods for processing deps.json files for patching
+        private void TempPatching(string patchDir)
+        {
+            var prefix = patchDir;
+            string[] filenames = new string[3] { "dotnet.deps.json", "MSBuild.deps.json", "NuGet.CommandLine.XPlat.deps.json" };
+            string[] fullnames = new string[3];
+            for(int i = 0; i < filenames.Length; i++) 
+            {
+                fullnames[i] = prefix + Path.DirectorySeparatorChar + filenames[i];
+            }
+            TempPatchingDepsJsonForNewlyAddedDlls(fullnames);
+
+        }
+        private void TempCopyNewlyAddedDlls(string[] dlls, string pathCopyFrom, string PathCopyTo)
+        {
+            foreach (var dll in dlls) 
+            {
+                var copyFromFullName = Path.Combine(pathCopyFrom, dll);
+                var copyToFullName = Path.Combine(PathCopyTo, dll);
+                File.Copy(copyFromFullName, copyToFullName);
+            }
+        }
+        private void TempPatchingDepsJsonForNewlyAddedDlls(string[] filePaths)
+        {
+            string nugetBuildTasksName = "NuGet.Build.Tasks/5.3.0-rtm.6251";
```

Comment:
Ugh.  What happens if this version changes?  Will tests fail?  Do we need to keep updating this until we have a permanent fix?

Reply:
Yes. If the version changes, tests fail. The permanent fix is https://github.com/NuGet/Home/issues/8570, and my understanding is, that's the same time when we're ready to ship the xplat verification.

Reply:
I'm not really sure what the action with this one is supposed to be?